tests/userTest.js

```diff
@@ -151,7 +151,7 @@ fdescribe('Main: Currently testing user management,', function () {
     })
   })
 
-  it('checking method : setUserAttribute of an existent user, allowed attribute', function (done) {
+  fit('checking method : setUserAttribute of an existent user, allowed attribute', function (done) {
```

Comment:
should it be reverted back? :thinking:

Reply:
Now since all the tests in this file can run, we can simply use `it` with `fdescribe`. I forgot to change those. It is fixed now